lib/Doctrine/ORM/Tools/EntityGenerator.php

```diff
@@ -277,10 +277,12 @@ public function <methodName>(<methodTypeHint>$<variableName>)
  * <description>
  *
  * @param <variableType> $<variableName>
+ *
+ * @return boolean TRUE if this collection contained the specified element, FALSE otherwise.
```

Comment:
I get it from https://github.com/doctrine/collections/blob/6c1e4eef75f310ea1b3e30945e9f06e652128b8a/lib/Doctrine/Common/Collections/Collection.php#L98

Also PSR-5 is draft. I'm not sure about change `boolean` to `bool`. Is it required?